infinispan-spring-boot-starter-remote/src/test/java/test/org/infinispan/spring/starter/remote/container/InfinispanContainer.java

```diff
@@ -42,7 +40,9 @@ protected void containerIsStarted(final InspectContainerResponse containerInfo)
             .statistics().jmxEnable();
 
       cacheManager = new RemoteCacheManager(configBuilder.build());
-      getCacheManager().administration().createCache(cacheName, "default");
+      if (cacheName != null) {
```

Comment:
Does it fix some bug that is worth  adding to the PR description?

Reply:
Not really, I just realized that if we don't pass any cacheName (that is not mandatory) I will have an error. but today we only use this infinispancontainer in a single place

Reply:
:+1: